frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -43,6 +46,22 @@ const handleError = (res: any, flags: string[], dispatch: Dispatch, cb: FeatureD
   }
 };
 
+export const detectRGW: FeatureDetector = async (dispatch) => {
+  let id = null;
+  const logicHandler = () =>
+    k8sList(StorageClassModel).then((data) => {
+      const provisioners = data.map((sc) => sc.provisioner);
+      if (provisioners.includes('openshift-storage.ceph.rook.io/bucket')) {
```

Comment:
```
const cephRGWProvisioner = 'openshift-storage.ceph.rook.io/bucket';
const checkRGWSCIsPresent = () => data.some((sc) => sc.provisioner === cephRGWProvisioner);

if(checkRGWSCIsPresent) { 
... 
}
```

Reply:
Probably use const file to define variable & `checkRGWSCIsPresent` in utils & can be used later for checks.